### PR TITLE
Update docs for pvpc_hourly_pricing integration

### PR DESCRIPTION
**Pull request by azogue**
## Proposed change

* Updated section about YAML configuration
* Update setup instructions for new config & options flow, supporting definition of new parameters for the updated sensor after the tariff change for electricity prices in Spain in 2021-06-01

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/51789
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/pvpc_hourly_pricing.markdown [hunk 1]
```diff
@@ -18,59 +18,33 @@ This sensor uses the official API to get the hourly price of electricity in Spai
...
-name:
-  description: Custom name for the sensor.
-  required: true
-  type: string
-tariff:
-  description: Contracted electric tariff.
-  required: true
-  default: discrimination
-  type: string
-{% endconfiguration %}
```
Comment (frenck) [inline source/_integrations/pvpc_hourly_pricing.markdown]: This change is not related to the parent PR and thus does not belong in this PR.

Should this be changed in our current documentation?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/pvpc_hourly_pricing.markdown (hunk 1 above)
Comment (azogue) [reply source/_integrations/pvpc_hourly_pricing.markdown]: Hi @frenck, you're totally right. I doubted with the PR destination (next or current), and how to change both in the same time, in sync with the parent PR.

The thing is the parent PR _could_ remove the `CONFIG_SCHEMA` and `async_setup`, as **they are already deprecated in 2021.6.0** (I think for 2021.7 should be removed, right?). Because, if not, then the change in the PR should be also reflected in the `CONFIG_SCHEMA` (the config_entry.data is changed), but I thought it was an overkill for something already deprecated, don't you agree?

But I have no problem in changing anything, splitting this one in 2, add another commit removing the yaml config in parent, open another in core for the yaml config removal... you name it ;-)

Comment (frenck) [reply source/_integrations/pvpc_hourly_pricing.markdown]: We generally deprecate the config schema for at least 2 release cycles.
Nevertheless, this is not related to the parent PR, so please move it out of the scope.

Comment (azogue) [reply source/_integrations/pvpc_hourly_pricing.markdown]: > We generally deprecate the config schema for at least 2 release cycles.

Ah, I thought it was just 1, sorry!

Then, instead of moving out the removal, what I need to do _here_ is to adapt it to the new schema that the parent PR introduces (so, editing it, not removing), and then would be inside the scope again, right?

Comment (azogue) [reply source/_integrations/pvpc_hourly_pricing.markdown]: @frenck, I just restored the YAML section, updating the fields to sync with the new schema introduced in the parent PR, I hope it's ok now

File: source/_integrations/pvpc_hourly_pricing.markdown [hunk 2]
```diff
@@ -53,10 +53,12 @@ PVPC Hourly Pricing allows manual configuration by adding a section to your `con
...
-  - name: PVPC manual ve
-    tariff: electric_car
-  - name: PVPC manual nocturna
-    tariff: discrimination
+  - name: PVPC
+    tariff: 2.0TD
+    power: 3.45
+    power_p3: 4.6
+  - name: PVPC-CYM
+    tariff: 2.0TD (Ceuta/Melilla)
```
Comment (frenck) [inline source/_integrations/pvpc_hourly_pricing.markdown]:
```suggestion
  - name: "PVPC"
    tariff: "2.0TD"
    power: 3.45
    power_p3: 4.6
  - name: "PVPC-CYM"
    tariff: "2.0TD (Ceuta/Melilla)"
```

File: source/_integrations/pvpc_hourly_pricing.markdown [hunk 3]
```diff
@@ -18,33 +18,33 @@ This sensor uses the official API to get the hourly price of electricity in Spai
...
+- `2.0TD (Ceuta/Melilla)`, for the cities of Ceuta and Melilla.
 
-- 1 period: `normal`, for the "Default PVPC tariff, (2.0 A)".
-- 2 periods: `discrimination`, for the "Efficiency 2 periods (2.0 DHA)", with 10h peak interval from 11:00 UTC to 21:00 UTC.
-- 3 periods: `electric_car`, for the "Electric vehicle tariff (2.0 DHS)", optimized for electric car owners to charge at night.
+Set also your contracted power (in kW) for the two power periods that apply with the new 2.0TD tariff
+(one for P1/P2 and the other one for the valley period, P3), to show your available electric power as a sensor attribute.
 
-The default is `discrimination`, with 2 periods, as it is usually the cheapest one for home consumers. 
+In case you did nothing after the tariff change on 2021-06-01, both powers are equal, and the same you had for your existing contract.
```
Comment (frenck) [inline source/_integrations/pvpc_hourly_pricing.markdown]:
```suggestion
In case you did nothing after the tariff change on 2021-06-01, both powers are equal, and the same you had for your existing contract.
```

File: source/_integrations/pvpc_hourly_pricing.markdown [hunk 4]
```diff
@@ -18,33 +18,33 @@ This sensor uses the official API to get the hourly price of electricity in Spai
...
-- 2 periods: `discrimination`, for the "Efficiency 2 periods (2.0 DHA)", with 10h peak interval from 11:00 UTC to 21:00 UTC.
-- 3 periods: `electric_car`, for the "Electric vehicle tariff (2.0 DHS)", optimized for electric car owners to charge at night.
+Set also your contracted power (in kW) for the two power periods that apply with the new 2.0TD tariff
+(one for P1/P2 and the other one for the valley period, P3), to show your available electric power as a sensor attribute.
 
-The default is `discrimination`, with 2 periods, as it is usually the cheapest one for home consumers. 
+In case you did nothing after the tariff change on 2021-06-01, both powers are equal, and the same you had for your existing contract. 
 
-You can add multiple sensors (up to 3, one per tariff) by adding them again through the integrations panel.
+You can add up to 2 sensors (one for each geographic zone) by adding them again through the integrations panel,
```
Comment (frenck) [inline source/_integrations/pvpc_hourly_pricing.markdown]:
```suggestion
You can add up to 2 sensors (one for each geographic zone) by adding them again through the integrations panel,
```

File: source/_integrations/pvpc_hourly_pricing.markdown [hunk 5]
```diff
@@ -18,33 +18,33 @@ This sensor uses the official API to get the hourly price of electricity in Spai
...
-To configure PVPC Hourly Pricing, you can set it up via the integrations panel in the configuration screen.
+To configure PVPC Hourly Pricing, set it up via the integrations panel in the configuration screen.
 
-Set a name for the price sensor (default is `sensor.pvpc`) and select one of the three tariffs, 
-corresponding to your contracted rate, according to the number of billing periods per day 
-(one / peak + valley / shifted peak + valley).
+Set a name for the price sensor (default is `sensor.pvpc`), and select one of the two available tariffs, 
+according to your geographic position in Spain: 
+- `2.0TD`, for the Peninsula, the Balearic Islands and the Canary Islands.
+- `2.0TD (Ceuta/Melilla)`, for the cities of Ceuta and Melilla.
```
Comment (frenck) [inline source/_integrations/pvpc_hourly_pricing.markdown]:
```suggestion
Set a name for the price sensor (default is `sensor.pvpc`), and select one of the two available tariffs,
according to your geographic position in Spain: 

- `2.0TD`, for the Peninsula, the Balearic Islands and the Canary Islands.
- `2.0TD (Ceuta/Melilla)`, for the cities of Ceuta and Melilla.
```

File: source/_integrations/pvpc_hourly_pricing.markdown [hunk 6]
```diff
@@ -18,33 +18,33 @@ This sensor uses the official API to get the hourly price of electricity in Spai
 
 Specifically, it shows the current __active energy invoicing price (FEU)__ in €/kWh, 
 which is the energy term hourly price applied in the consumers' electrical bill 
-with a contracted power not exceeding 10 kW and which are under the PVPC 
+with a contracted power not exceeding 15 kW and which are under the PVPC
```
Comment (frenck) [inline source/_integrations/pvpc_hourly_pricing.markdown]:
```suggestion
with a contracted power not exceeding 15 kW and which are under the PVPC
```

File: source/_integrations/pvpc_hourly_pricing.markdown [hunk 7]
```diff
@@ -18,33 +18,33 @@ This sensor uses the official API to get the hourly price of electricity in Spai
...
-with a contracted power not exceeding 10 kW and which are under the PVPC 
+with a contracted power not exceeding 15 kW and which are under the PVPC 
 (Voluntary Price for Small Consumer).
 
 It includes the energy term of the access tolls, the charges and the production cost. It does not include taxes.
-The hourly prices are the same throughout the Spanish territory regardless of the time zone.
-
-It can be set up via the integrations panel in the configuration screen.
+The hourly prices and energy periods are the same throughout the Spanish territory regardless of the time zone, 
+except for the cities of Ceuta and Melilla, where they are slightly different.
```
Comment (frenck) [inline source/_integrations/pvpc_hourly_pricing.markdown]:
```suggestion
The hourly prices and energy periods are the same throughout the Spanish territory regardless of the time zone,
except for the cities of Ceuta and Melilla, where they are slightly different.
```

**Review by frenck — APPROVED** [review]
Thanks, @azogue 👍
